README.md

```diff
@@ -249,74 +335,148 @@ podTemplate(yaml: """\
       - name: busybox
         image: busybox
         command:
-        - cat
-        tty: true
-    """.stripIndent()) {
+        - sleep
+        args:
+        - 99d
+    '''.stripIndent()) {
     node(POD_LABEL) {
       container('busybox') {
         echo POD_CONTAINER // displays 'busybox'
-        sh "hostname"
+        sh 'hostname'
       }
     }
 }
 ```
 
-You can use [`readFile`](https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-readfile-code-read-file-from-workspace) or [`readTrusted`](https://jenkins.io/doc/pipeline/steps/coding-webhook/#readtrusted-read-trusted-file-from-scm) steps to load the yaml from a file.  It is also accessible from this plugin's configuration panel in the Jenkins console.
+You can use [`readFile`](https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#readfile-read-file-from-workspace) or [`readTrusted`](https://jenkins.io/doc/pipeline/steps/coding-webhook/#readtrusted-read-trusted-file-from-scm) steps to load the yaml from a file.  It is also accessible from this plugin's configuration panel in the Jenkins console.
```

Comment:
```suggestion
You can use [`readFile`](https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#readfile-read-file-from-workspace) or [`readTrusted`](https://jenkins.io/doc/pipeline/steps/coding-webhook/#readtrusted-read-trusted-file-from-scm) steps to load the yaml from a file.
```

No idea what that referred to.

Would be a good place to mention the `yamlFile` Declarative syntax.